util/rbac/rbac.go

```diff
@@ -472,6 +472,12 @@ func loadPolicyLine(line string, model model.Model) error {
 		return fmt.Errorf("invalid RBAC policy: %s", line)
 	}
 
+	for i := 0; i < len(tokens); i++ {
```

Comment:
I wonder if a role could contain a space. Like `p, some spacey role, applications, *, myapp/*, allow

Reply:
I made the assumption that not indeed.

Reply:
Want to limit this check to everything besides the role field, so it's an easy-merge?

Reply:
will push a change

Reply:
Can't we just assume that there are two types of policies right now:

* Grants (`g`), which has two fields (the grantee and the role granted)
* Policy (`p`), which has five fields (the role, the resource type, the verb, the resource name pattern and the action)

So wouldn't it make more sense to evaluate the policy type and check whether it has the right amount of fields set?

Reply:
I think we might already do that. But the fields are split on commas, not spaces.

Reply:
I ment, instead of checking whether a field contains white space, couldn't we just check for `len(tokens)` according to the type of policy (`g` or `p`) of the currently evaluated record?

Reply:
Oh, sure, since we wouldn't hit the right number of tokens when there's a comma missing. I do like that better.

Reply:
Will push the fix torward that approach then, i wasn't sure if there was special cases that i was not was about.

Reply:
@alexmt Are you aware of RBAC policy definitions that have a different pattern, e.g. a variable number of fields in a permission definition?